tests/test_U19/test_procedures.py

```diff
@@ -0,0 +1,248 @@
+    )
+    def test_run_job(self, mock_download_procedure):
+        """Test run_job method."""
+
+        with open(EXAMPLE_DOWNLOAD_PROCEDURE, "r") as f:
+            mock_download_procedure.return_value = json.load(f)
+
+        etl = U19Etl(self.example_job_settings)
+        job_response = etl.run_job()
+
+        actual_output = json.loads(job_response.data)
+
+        self.assertEqual(self.example_output, actual_output)
+
+    @patch(
+        "aind_metadata_mapper.U19.procedures.U19Etl.download_procedure_file"
+    )
+    def test_extract(self, mock_download_procedure):
+        """Test extract method."""
+
+        with open(EXAMPLE_DOWNLOAD_PROCEDURE, "r") as f:
+            mock_download_procedure.return_value = json.load(f)
+
+        etl = U19Etl(self.example_job_settings)
+        extracted = etl._extract(self.example_job_settings.subject_to_ingest)
+
+        self.assertEqual(
+            extracted["subject_id"],
+            self.example_job_settings.subject_to_ingest,
+        )
+
+    def test_transform(self):
+        """Test transform method."""
+
+        etl = U19Etl(self.example_job_settings)
+        etl.load_specimen_procedure_file()
+
+        with open(EXAMPLE_DOWNLOAD_PROCEDURE, "r") as f:
+            extracted = json.load(f)
+
+        transformed = etl._transform(
+            extracted, self.example_job_settings.subject_to_ingest
+        )
+
+        self.assertEqual(
+            len(transformed.specimen_procedures),
+            len(
+                construct_new_model(
+                    self.example_output, Procedures, True
+                ).specimen_procedures
+            ),
+        )
+
+    @patch("aind_metadata_mapper.U19.procedures.U19Etl._transform")
+    def test_load(self, mock_transform):
+        """Test load method."""
+
+        mock_transform.return_value = construct_new_model(
+            self.example_output, Procedures, True
+        )
+
+        etl = U19Etl(self.example_job_settings)
+        transformed = etl._transform(
+            self.example_job_settings.subject_to_ingest
+        )
+
+        job_response = etl._load(
+            transformed, self.example_job_settings.output_directory
+        )
+
+        actual_output = json.loads(job_response.data)
+
+        self.assertEqual(self.example_output, actual_output)
+
+    def test_find_sheet_row(self):
+        """Test find_sheet_row method."""
+
+        etl = U19Etl(self.example_job_settings)
+        etl.load_specimen_procedure_file()
+        row = etl.find_sheet_row(self.example_job_settings.subject_to_ingest)
+
+        self.assertTrue(row is not None)
+
+    @patch("requests.get")
+    def test_download_procedure_file(self, mock_requests):
+        """Test download_procedure_file method."""
+
+        with open(EXAMPLE_DOWNLOAD_RESPONSE, "r") as f:
+            example_download_response = json.load(f)
+            mock_requests.return_value.json.return_value = (
```

Comment:
This isn't actually setting the mock and is making a request. It's better to load all the json files in the setup class method. Since your src code is also doing a lot with the requests status code, you'll need to set a status code also.